src/main/java/org/opensearch/sdk/sample/helloworld/ExampleCustomSettingConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk.sample.helloworld;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Setting.RegexValidator;
+import org.opensearch.common.settings.Setting.Property;
+
+/**
+ * {@link ExampleCustomSettingConfig} contains the custom settings value and their static declarations.
+ */
+public class ExampleCustomSettingConfig {
+    private static final String FORBIDDEN_REGEX = "forbidden";
+    private final String validated;
```

Comment:
Code looks great except for this variable which is never set. You have a getter for it, though! 

Both can probably be removed.

Reply:
ACK